.github/workflows/create-cherry-pick-pr.yml

```
@@ -13,19 +13,27 @@ on:
         description: 'PR # Associated with Cherry Pick'
         required: true
 
-
 jobs:
   cherry-pick:
     runs-on: ubuntu-latest
-
+    permissions:
+      contents: write
+      pull-requests: write
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v4
-
-    - name: Create Cherry Pick PR
-      id: create-cherry-pick-pr
-      shell: bash
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      run: |
-        ./scripts/create-cherry-pick-pr.sh ${{ github.event.inputs.branch_name }} ${{ github.event.inputs.commit_hash }} ${{ github.event.inputs.PR_number }}
\ No newline at end of file
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Get Node.js version
+        id: nvm
+        run: echo "NODE_VERSION=$(cat .nvmrc)" >> "$GITHUB_OUTPUT"
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ steps.nvm.outputs.NODE_VERSION }}
```

```suggestion
      - uses: actions/setup-node@v4
        with:
          node-version-file: .nvmrc
```

https://github.com/actions/setup-node?tab=readme-ov-file#usage